requirements/base.txt

```diff
@@ -28,6 +28,7 @@ pathlib==1.0.1
 python-dateutil==2.6.0
 python-levenshtein==0.12.0
 pytz==2017.2
+requests=2.18.0
```

Comment:
`==` and could you add a comment about brokenness fix, I don't think we want to manage this when it's finally fixed.